dolphin/api/v1/access_info.py

```diff
@@ -37,8 +42,38 @@ def show(self, req, id):
 
         return self._view_builder.show(access_info)
 
+    @validation.schema(schema_access_info.update)
+    def update(self, req, id, body):
+        """Update storage access information."""
+        ctxt = req.environ.get('dolphin.context')
+        # Get existing access_info and storage from DB
+        try:
+            access_info_present = db.access_info_get(ctxt, id)
+            storage_present = db.storage_get(ctxt, id)
+            access_info_updated_dict = access_info_present.to_dict()
+            access_info_updated_dict.update(body)
+            # Discover storage with new access info
+            storage = self.driver_api.discover_storage(ctxt, access_info_updated_dict)
+            if storage['serial_number'] != storage_present.serial_number:
+                reason = (_("Existing storage serial Number is not matching with th new storage serial number: '%s'  "
```

Comment:
we may need to follow the python conventions. Ex: max 79 chars in a single line.

Reply:
used Pycharm code formatting tool only.. If reformat it say "Content is already properly formatted" :)

Reply:
![python_convention](https://user-images.githubusercontent.com/41313813/81683964-31bfec00-9474-11ea-8611-78e2928188a1.png)

I am not sure about pycharm, but python.org has this convention

Reply:
Usual guideline is to have 79 chars

Reply:
done